src/components/validators/project-name-available-validator.js

```diff
@@ -28,6 +31,10 @@
 
             return deferred.promise;
           };
+
+          scope.$watch("organizationId", function() {
+            ngModel.$validate();
```

Comment:
From testing, it was needed.

So say **My Organization** has i.e. **My Project** on the list, whereas **My Other Organization** doesn't (other projects or empty list).

Having **My Other Organization** selected, and afterwards entering **My Project** in the `project_name` field would of course result as valid.

Then if changing to `My Organization` instead would - without the watch part - not revalidate automatically.

This it also the case used in [this example for password compare](http://odetocode.com/blogs/scott/archive/2014/10/13/confirm-password-validation-in-angularjs.aspx).